recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - xeus >=3.0.0,<4
     - xtl >=0.7.7,<0.8
     - pugixml
-    - cpp-argparse >=2.9,<3
+    - cpp-argparse >=2.9,<=3.0
```

Comment:
I think that including 3.0 in the range is fine, but I would prefer using `<3.1` to allow for patch version upgrades.